docs/docs/usage.md

```diff
@@ -2,4 +2,93 @@
 title: Usage
 ---
 
-*Coming soon...*
+## General usage
+In general, you call the cli like so: <br>
+`ccom [options] <input>`
+
+For example a call could look like this:
+`ccom --data ./data-file.json --out-file ./output-file.yml ./input-file.yml`
+
+For customization, you have following cli options:
+
+| Option                              | Shortcut       | Description                                                                           | Default |
+|-------------------------------------|----------------|---------------------------------------------------------------------------------------|---------|
+| --benchmark <number>                | -b <no>        | Execute compiler benchmarks. n is the number of benchmark runs                        | 0       |
```

Comment:
I can't verify that `<` is working for the docs. Can you make sure it is printed correctly?

Reply:
Yes, the documentation was also affected